asterius/rts/rts.messages.mjs

```diff
@@ -0,0 +1,11 @@
+export class Messages {
+    constructor(memory) {
+        this.memory = memory;
+        Object.seal(this);
+    }
+
+    debugBelch2(fmt, arg) {
+        console.error(this.memory.strLoad(fmt), this.memory.strLoad(arg));
```

Comment:
`console.error` logs to the `node` process's standard error, keep in mind that we have our own `stderr` and that custom `stderr` buffer is checked by `ghc-testsuite`. So do make sure we log to our own `stderr` as well.

Reply:
Done.